lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
Agreed on there not being a big gain there. I was on the fence about even mentioning it.

There will be a slightly higher improvement in node land from what I recall, but less than 10% improvement from previous tests with similar changes.